packages/contracts-core/contracts/test/Home.t.sol

```diff
@@ -115,4 +114,78 @@ contract HomeTest is NomadTestWithUpdaterManager {
         home.update(oldRoot, newRoot, sig);
         assertEq(newRoot, home.committedRoot());
     }
+
+    function test_updateRejectFailedState() public {
+        bytes memory messageBody = "";
+        uint32 destinationDomain = remoteDomain;
+        bytes32 recipient = bytes32(uint256(uint160(vm.addr(1505))));
+        home.dispatch(destinationDomain, recipient, messageBody);
```

Comment:
you don't need to send a message in order to `test_improperUpdate` - you can remove these lines to streamline the code